components/security/authorization/pom.xml

```diff
@@ -47,6 +47,18 @@
       <artifactId>xwiki-platform-users-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>phenotips-constants</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
```

Comment:
My test was failing because it couldn't find the class. I'm honestly not sure why I had to include the dependency even though I never explicitly use it.